src/components/Table/EditableDraggableTable.js

```diff
@@ -132,11 +132,12 @@ class EditableDraggableTable extends React.Component {
   };
 
   getActionButton = (type, additionalData, id) => {
+    const renderConfig = this.getRenderConfig(additionalData);
     switch (type) {
       case DELETE_ACTION:
         return (
           <MenuItem id={id} key={id} onSelect={() => this.crudController.onDelete(additionalData)}>
-            Remove Disk
+            {renderConfig.text}
```

Comment:
please make it more configurable under renderConfig.actions. Then you can also pass it as a parameter